skypy/galaxy/spectrum.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import numpy as np
+import astropy.units as u
```

Comment:
```suggestion
from astropy import units
```
Strong preference not to import units as `u` please it's really not a commonly recognised convention and leads to some very opaque names e.g. `u.AA`. Same in tests as well. Otherwise this all looks good.

Reply:
Using `u` is following astropy best practises. And personally, I find `u.Mpc` much more quickly parseable than `units.Mpc`.

Reply:
It is also at odds with our own style of avoiding abbreviations in favour of code clarity wherever practical

Reply:
My feeling on this is that python conventions > numpy conventions > astropy conventions > our conventions but I am happy to change it if that's the official line from here on out.

Reply:
The problem I see is that while it is used as a consistent convention by the astropy developers within the astropy codebase and documentation, I doubt that it is recognised in the wider community of astropy and skypy users, certainly not to the same level as `import numpy as np` (which I tolerate).